src/simplewallet/simplewallet.cpp

```diff
@@ -969,12 +989,7 @@ bool simple_wallet::finalize_multisig(const std::vector<std::string> &args)
     return true;
   }
 
-  const auto orig_pwd_container = get_and_verify_password();
-  if(orig_pwd_container == boost::none)
-  {
-    fail_msg_writer() << tr("Your original password was incorrect.");
-    return true;
-  }
```

Comment:
Doesn't the moving of this code section to above have no effect? If so, it seems better to revert this to minimize changes.